src/Bicep.Core/Emit/ExpressionConverter.cs

```diff
@@ -24,24 +24,20 @@ namespace Bicep.Core.Emit
     public class ExpressionConverter
     {
         private readonly EmitterContext context;
+        private readonly ExpressionBuilder expressionBuilder;
 
-        private readonly ImmutableDictionary<LocalVariableSymbol, LanguageExpression> localReplacements;
+        private readonly ImmutableDictionary<LocalVariableSymbol, Expression> localReplacements;
```

Comment:
Is this purely for better organization of the Expression subclasses? Will LanguageExpression still be used?

Reply:
See my other comment regarding the purpose of this PR.

With that context, the `Expression` is actually the base class for the intermediate representation. I'm wondering if I should rename it to clarify, as this is a bit confusing. (any naming suggestions - let me know).

So when doing code generation, eventually we will have something that looks like:
`SyntaxBase -> Expression -> JSON/LanguageExpression`